macros/index.sql

```diff
@@ -0,0 +1,5 @@
+{% macro index(this, column) %}
```

Comment:
can we move this to `macros/constraints/index.sql`? I'm unsure what other kinds of macros will show up here in the future, but I'd like to start building some sort of structure. I'm definitely open to other ideas too!

Reply:
The filepath doesn't actually matter -- it's strictly for organization